src/pages/views/overview/components/dashboardWidgetBase.tsx

```diff
@@ -606,23 +612,26 @@ class DashboardWidgetBase extends React.Component<DashboardWidgetProps> {
   };
 
   private getTitle = () => {
-    const { t, titleKey } = this.props;
+    const { titleKey } = this.props;
+    const intl = createIntlEnv(getLocale());
 
-    const today = new Date();
-    const month = getMonth(today);
-    const endDate = format(today, 'Do');
-    const startDate = format(startOfMonth(today), 'Do');
+    // const today = new Date();
+    // const month = getMonth(today);
+    // const endDate = format(today, 'Do');
+    // const startDate = format(startOfMonth(today), 'Do');
 
-    return t(titleKey, { endDate, month, startDate });
+    return intl.formatMessage(titleKey);
```

Comment:
This doesn't look correct. Seems to be missing some dates

Reply:
Never mind, probably left over from a redesign. We don't show dates in the card titles